noxfile.py

```diff
@@ -322,7 +332,8 @@ def upload_release(session):
             f"Remove dist/ and run 'nox -s build-release -- {version}'"
         )
     # Sanity check: Make sure the files are correctly named.
-    distfile_names = map(os.path.basename, distribution_files)
+    # Ignore typing bug: https://github.com/python/mypy/issues/9864
+    distfile_names = map(os.path.basename, distribution_files)  # type: ignore
```

Comment:
Can this be worked around with

```python
distfile_names = (os.path.basename(fn) for fn in distribution_files)
```

Reply:
Yup, that works. Handled in #9411.

Does the project have a styling preference on `map()` vs generator expressions in general, or no preference?